sigle/src/modules/publicHome/components/FixedHeader.tsx

```diff
@@ -0,0 +1,188 @@
+import { styled } from '../../../stitches.config';
+import {
+  Box,
+  Button,
+  Container,
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogTitle,
+  Flex,
+  Typography,
+} from '../../../ui';
+import {
+  useGetUserFollowing,
+  useUserFollow,
+  useUserUnfollow,
+} from '../../../hooks/appData';
+import { useFeatureFlags } from '../../../utils/featureFlags';
+import { useAuth } from '../../auth/AuthContext';
+import { generateAvatar } from '../../../utils/boringAvatar';
+import { SettingsFile } from '../../../types';
+import { HeaderDropdown } from '../../layout/components/HeaderDropdown';
+import { useState } from 'react';
+import Link from 'next/link';
+
+const Header = styled('header', Container, {
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'space-between',
+  width: '100%',
+});
+
+const ImageContainer = styled('div', {
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  overflow: 'hidden',
+  width: 24,
+  height: 24,
+  br: '$1',
+});
+
+interface FixedHeaderProps {
+  userInfo: { username: string; address: string };
+  settings: SettingsFile;
+  // scrollY: number;
+}
+
+export const FixedHeader = ({
+  userInfo,
+  settings,
+}: // scrollY,
+FixedHeaderProps) => {
+  const { user } = useAuth();
+  const { isExperimentalFollowEnabled } = useFeatureFlags();
+  const { data: userFollowing } = useGetUserFollowing({
+    enabled: !!user && userInfo.username !== user.username,
+  });
+  const { mutate: followUser } = useUserFollow();
+  const { mutate: unfollowUser } = useUserUnfollow();
+  const [showLoginPromptDialog, setShowLoginPromptDialog] = useState(false);
+
+  const handleShowLoginPrompt = () => setShowLoginPromptDialog(true);
+  const handleCloseLoginPrompt = () => setShowLoginPromptDialog(false);
+
+  const handleFollow = async () => {
+    if (!userFollowing) return;
+    followUser({ userFollowing, address: userInfo.address });
+  };
+
+  const handleUnfollow = async () => {
+    if (!userFollowing) {
+      return;
+    }
+    unfollowUser({ userFollowing, address: userInfo.address });
+  };
+
+  const userAddress =
+    user?.profile.stxAddress.mainnet || user?.profile.stxAddress;
+
+  const isFollowingUser =
+    userFollowing && !!userFollowing.following[userInfo.address];
+
+  const scrollY = typeof window !== 'undefined' ? window.scrollY : 0;
+
+  return (
+    <Header
+      css={{
+        backgroundColor: '$gray1',
+        position: 'sticky',
+        top: 0,
+        transform: scrollY > 228 ? 'none' : 'translateY(-200px)',
```

Comment:
as discussed with Quentin, we decided it would be good to bring the header in as the user is scrolling past the profile image & follow button (around 228) to have a consistent CTA